### PR TITLE
Set proper python path for dawn examples

### PR DESCRIPTION
**Pull request by twicki**
## Technical Description

Fixes the path in the jenkins build script to actually run the python tests. Also the build-script exits on error now.

#### Enhances

This enhances testing since the functionality that is developed but not used in the default configuration is tested like this. 

## Dependencies

This PR is independent

### CODE REVIEW

**Comment by twicki** [comment]
launch jenkins

**Comment by twicki** [comment]
launch jenkins

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -74,7 +74,7 @@ option(DAWN_PYTHON_EXAMPLES "Build the python examples of generating HIR" OFF)
 
 if( DAWN_BUNDLE_PYTHON )
   if(NOT DEFINED PYTHON_EXECUTABLE)
-    find_package(PythonInterp 3.4 REQUIRED)
+    find_package(PythonInterp 3.5 REQUIRED)
```
Comment (lukasm91) [inline bundle/CMakeLists.txt]: Do we really need Python >=3.5? Maybe it should be enough to do
```
find_package(PythonInterp 3 REQUIRED)
```

**Comment by twicki** [comment]
launch jenkins

File: bundle/CMakeLists.txt (hunk 1 above)
Comment (twicki) [reply bundle/CMakeLists.txt]: https://github.com/MeteoSwiss-APN/dawn/blob/f49685c38a60370f91adbf64e548a4bfc90ec599/python/dawn/sir.py#L19

is a python 3.5 feature

**Comment by twicki** [comment]
launch jenkins

**Comment by twicki** [comment]
launch jenkins

**Pull request merged**